doc/guide/dynamics/dynamics-floquet.rst

```diff
@@ -92,8 +93,7 @@ Consider for example the case of a strongly driven two-level atom, described by
 
 In QuTiP we can define this Hamiltonian as follows:
 
-.. plot::
-   :context: reset
+.. code-block:: python
```

Comment:
Was there some issue with running this block?

Reply:
No, but elsewhere when we use plot, we don't add the `>>> `. Also in the next block, there are output mixed in and I was hoping changing something else would be able to generate them, but no luck.